css/comp.css

```diff
@@ -0,0 +1,115 @@
+@import "./core.css";
+@import "./docPage.css";
+@import "./util.css";
+
+.exapmle-container {
+  background-color: white;
+  border: 0.1rem solid var(--dark);
+  padding: 1rem;
+  border-radius: 1rem;
+}
+
+.elm-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: start;
+  margin-bottom: 3rem;
+  align-items: center;
+}
+
+/* ---------------------- Avatar -----------------------*/
+
+.avatar {
+  height: 4rem;
+  width: 4rem;
+  border-radius: 50%;
+  margin-right: 1rem;
+}
+
+.avatar-sq {
+  height: 4rem;
```

Comment:
don't repeat the same property height, width, margin-right.

Reply:
I did it initially but then it would increase the amount of code that i someone would have to write in div.
for example: in the above case one can write className="avatar" or  className="avatar-sq" for a basic template but if i made it a utility class then one would have to write className="avatar avatar-sq" so in order to counter that i made these two different classes.